activiti-api-process-model/src/main/java/org/activiti/api/process/model/events/BPMNActivityEvent.java

```diff
@@ -12,7 +12,9 @@
 
         ACTIVITY_CANCELLED,
 
-        ACTIVITY_COMPLETED
+        ACTIVITY_COMPLETED,
+        
+        ACTIVITY_SIGNALED
```

Comment:
@CTI777 in terms of naming I'm not very confortable with `ACTIVITY_SIGNALED`, even if this is the name used by the internal event, it suggests that this is related to an activity, but in reality it's related to a BPMN event (catch signal event, that conceptually it's not an activity). So maybe `SIGNAL_CAUGTH` or `SIGNAL_RECEIVED` would be better names.
@CTI777 before you change this I'd like to have @salaboy's input as well, as I don't want to make you change names across the repositories twice :)

Reply:
@CTI777 I've checked with @salaboy. Let's go with `SIGNAL_RECEIVED`.
Note that you also need to move the enum entry outside of `BPMNActivityEvent.ActivityEvents` you should create a dedicated class for signals, similar to what we have for [SequenceFlow](https://github.com/Activiti/activiti-api/blob/develop/activiti-api-process-model/src/main/java/org/activiti/api/process/model/events/BPMNSequenceFlowTakenEvent.java)